ruby-challenges.rb

```diff
@@ -23,6 +40,18 @@
 # Expected output: 'bby Rd'
 
 # Pseudo code:
+# Define a method called no_vowels
+# Input: string
+# Output: string
+# Use .delete! to remove the vowels from the string
+
+def no_vowels str
+    str.delete! "aeiou"
```

Comment:
Ooo, you really mean it with that delete!  Something to think about, what happens with uppercased vowels?